bls/src/main/java/tech/pegasys/teku/bls/hashToG2/Helper.java

```diff
@@ -54,14 +58,25 @@ static boolean isOnCurve(JacobianPoint p) {
   /**
    * Tests whether the given point lies in the G2 group.
    *
-   * <p>Once the GLV patent has expired we can replace the qChain with the endomorphism here:
-   * https://eprint.iacr.org/2019/814.pdf
-   *
    * @param p a JacobianPoint
    * @return true if the point is in G2, false otherwise
    */
   static boolean isInG2(JacobianPoint p) {
```

Comment:
(nit) Might be a good idea to make the methods that are directly used in production `public`, and either mark the other methods `private` or keep them package-private and annotate them with `@VisibleForTesting`.  This just makes it easier to see which methods are internal-only vs used externally.

Reply:
I'll be doing another pass through everything for the v0.12 update, and will consider holding my nose and adding `@VisibleForTesting` to everything. Very little can be marked `private` currently as just about every method has a corresponding unit test.

This is another reason I try not to write too much Java 😛.